FAO Publications.js

```diff
@@ -245,11 +249,11 @@ function scrape(doc, url) {
 			newItem.place = 'Rome, Italy';
 		}
 		// If there's no author, use 'FAO' as author.
-		if (!newItem.creators) {
+		if (newItem.creators.length === 0) {
```

Comment:
!newItem.creators doesn't work. Maybe because creators is an array?

An example with no author in the webpage is the last test case:
http://www.fao.org/publications/card/en/c/5014f143-be17-4b58-b90e-f1c6bef344a0/

Reply:
Yes, an empty array isn't false in JS. But it can just be `if (!newItem.creators.length) {`.

Reply:
Changed.